libraries/resource_providers/api200/ethernet_network_provider.rb

```diff
@@ -13,6 +13,8 @@ module OneviewCookbook
   module API200
     # EthernetNetworkProvider
     class EthernetNetworkProvider < ResourceProvider
+      include API200::ConnectionTemplateProvider::ConnectionTemplateHelper
+
```

Comment:
Let's try again:

You still need to put `require_relative 'conection_template_provider'`

Reply:
ok, ok. :D